nginx/tox.ini

```diff
@@ -28,3 +28,8 @@ setenv =
     1.12: NGINX_IMAGE=nginx:1.12
     1.13: NGINX_IMAGE=nginx:1.13
     vts: NGINX_IMAGE=datadog/docker-library:nginx-vts
+
+[testenv:latest]
+setenv =
+    DDEV_SKIP_GENERIC_TAGS_CHECK=true
```

Comment:
Why is this set to `true` for latest versions?

Reply:
I'm not really sure, but we have it for all the other fixed versions: https://github.com/DataDog/integrations-core/blob/d2a7f0737e396f397fb822c1c6cbe6de67385b01/nginx/tox.ini#L27

Reply:
This flag is to skip some validation. We should not add it to integrations that do not have it already. AI-1633 tracks the work to remove it from the integrations that have it.